examples/minimal/packages/client-react/.eslintrc

```diff
@@ -1,4 +1,5 @@
 {
+  "root": true,
```

Comment:
for context - why do we need this here but not in the `.eslint` in the react template?